### ISSUE TITLE
Can PANTHEON_MACHINE_TOKEN be preferred over PANTHEON_TOKEN when running ddev task sync?

### ISSUE THREAD

**Issue by davereid** [Lullabot/drainpipe#358]
Most of DDEV uses `TERMINUS_MACHINE_TOKEN` in it's examples, and at another client project where we had a standardized DDEV config across all developers, we used `PANTHEON_MACHINE_TOKEN`. I'm trying to understand why the environment variable is named `PANTHEON_TOKEN` and not the more standard examples we've seen in other projects. Could we prefer a more standard name but provide support for the old value?

**Comment by mrdavidburns** [comment]
We probably need to propose this change in drainpipe.
https://github.com/Lullabot/drainpipe/blob/d2181df595d551db567186c418307c25efbe735e/tasks/pantheon.yml#L10

**Comment by deviantintegral** [comment]
👍 to this! We can also do this as a fallback to still support PANTHEON_TOKEN until the next major version.

**Comment by davereid** [comment]
Yup, we used a fallback to the previous environment name so it worked automatically for existing installs.
